src/libraries/Microsoft.CSharp/src/ILLink/ILLink.Suppressions.xml

```diff
@@ -0,0 +1,143 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<linker>
+  <assembly fullname="Microsoft.CSharp, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
```

Comment:
Omitting the version number should work as well. I don't think we have to use it here

Reply:
Done. I assume this is useful so we don't have to update it in future versions (if the suppressions files are still in the codebase i.e not resolved). If so, would it make sense to have an option when using `--generate-warning-suppressions` to not emit the version number? Not sure if many folks beyond the libraries team use the feature, and this is perhaps the last time we'll generate these files.

Reply:
What about `Culture=neutral`? That seems unnecessary as well.

Reply:
Removed.